### PR TITLE
Development

### PR DESCRIPTION
**Pull request by htuerker**
Project finished.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,484 @@
...
+            <i class="fas fa-directions"></i> &nbsp;
+            <i class="far fa-bookmark"></i> &nbsp;
+            <i class="far fa-comment-alt"></i>
+        </div>
+    </div>
+
+    <div class="main-content">
+        <header>
+            <h1>
+                <div class="article-title">Space Ripples Reveal Big Bang’s Smoking Gun
```
Comment (heyapricot) [inline index.html:46]: Element `div` not allowed as child of element `h1` in this context.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,484 @@
...
+                    few, meeting with Dr. Guth, who is now a professor at Massachusetts Institute of Technology (as is
+                    his son, Larry, who was sleeping that night in 1979), in his office last week.</p>
+                <p>“It was a very special moment, and one we took very seriously as scientists,” said Dr. Kovac, who
+                    chose his words as carefully as he tended his radio telescopes.</p>
+                <p>Andrei Linde of Stanford, a prolific theorist who first described the most popular variant of
+                    inflation, known as chaotic inflation, in 1983, was about to go on vacation in the Caribbean last
+                    week when Chao-Lin Kuo, a Stanford colleague and a member of Dr. Kovac’s team, knocked on his door
+                    with a bottle of Champagne to tell him the news.</p>
+
+                <iframe width="630" height="358" src="https://www.youtube.com/embed/ZlfIVEy_YOA" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
```
Comment (heyapricot) [inline index.html:209]: The frameborder attribute on the iframe element is obsolete. Use CSS instead.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,484 @@
...
+                    <br>
+                    <span style="font-size:12px;">Jan, 2</span>
+                </div>
+            </div>
+        </div>
+        <div class="trends">
+            <hr>
+            <strong style="font-size:18px;">Trending</strong>
+            <ul>
+                <hr>
```
Comment (heyapricot) [inline index.html:348]: Element `hr` not allowed as child of element `ul` in this context.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,484 @@
...
+    </div>
+    <div class="footer">
+        <hr>
+        <div class="footer-header">
+            <img src="./assets/logo.png" alt="">
+            <small><a href="#"> Go to Homepage >> </a></small>
+        </div>
+        <div class="footer-links">
+            <ul class="link-column">
+                <li>NEWS</li><br>
```
Comment (heyapricot) [inline index.html:383]: Element `br` not allowed as child of element `ul` in this context

File: index.html [hunk 5]
```diff
@@ -0,0 +1,484 @@
...
+                    <li>Tech</li>
+                    <li>Science</li>
+                    <li>Sports</li>
+                    <li>Obituaries</li>
+                    <li>Today's Paper</li>
+                    <li>Corrections</li>
+            </ul>
+            <ul class="link-column">
+                    <li>NEWS</li><br>
+                    <strong><li>Home Page</li>
```
Comment (heyapricot) [inline index.html:451]: Element `strong` not allowed as child of element `ul` in this context.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,484 @@
...
+                <a href="#">Privacy</a>
+                <a href="#">Terms of Service</a>
+                <a href="#">Terms of Sale</a>
+                <a href="#">Site Map</a>
+                <a href="#">Help</a>
+                <a href="#">Subscriptions</a>
+            </div>
+        </div>
+    </div>
+    </div>
```
Comment (heyapricot) [inline index.html:481]: What is this closing?

File: css/styles.css [hunk 7]
```diff
@@ -0,0 +1,424 @@
...
+    background-color: rgb(56, 80, 99);
+}
+
+
+.article-links{
+    width:50%;
+    float:right;
+    text-align:right;
+    position:relative;
+    padding-left:0px;
```
Comment (heyapricot) [inline css/styles.css:108]: You can omit the `px` when value is 0

File: css/styles.css [hunk 8]
```diff
@@ -0,0 +1,424 @@
...
+    border-radius: 0;
+}
+.article-links > i:last-child:hover {
+    color: gray;
+}
+
+.article-links > i:nth-child(5) {
+    font-size:24px;
+    border: none;
+    border: 1px solid white;
```
Comment (heyapricot) [inline css/styles.css:139]: This overwrites the property on line 138

File: css/styles.css [hunk 9]
```diff
@@ -0,0 +1,424 @@
...
+    background-color: white;
+    z-index: 9999;
+    justify-content: space-between;
+    border-bottom: 1px solid rgb(204, 201, 201);
+    padding-top: 5px;
+    padding-bottom: 10px;
+}
+
+.topbar > * {
+    padding: 0px 10px;
```
Comment (heyapricot) [inline css/styles.css:228]: You can omit the `px` when value is 0

File: css/styles.css [hunk 10]
```diff
@@ -0,0 +1,424 @@
...
+    padding: 10px 10px;
+}
+
+.topbar > .links {
+    display: flex;
+    justify-content: space-between;
+}
+
+.topbar > .links {
+    padding-left:0px;
```
Comment (heyapricot) [inline css/styles.css:262]: You can omit the `px` when value is 0

File: css/styles.css [hunk 11]
```diff
@@ -0,0 +1,424 @@
...
+.article-footer {
+    color: gray;
+    font-size: 14px;
+    font-weight: lighter;
+}
+
+.topbar {
+    width: 100%;
+    height: 30px;
+    display: flex;
```
Comment (heyapricot) [inline css/styles.css:217]: Since you are injecting the display: flex with javascript, do you need it on your class?

File: index.html [hunk 12]
```diff
@@ -0,0 +1,484 @@
...
+                    realms of time and space and energy to science and speculation.
+                </p>
+                <p>Confirming inflation would mean that the universe we see, extending 14 billion light-years in space
+                    with its hundreds of billions of galaxies, is only an infinitesimal patch in a larger cosmos whose
+                    extent, architecture and fate are unknowable. Moreover, beyond our own universe there might be an
+                    endless number of other universes bubbling into frothy eternity, like a pot of pasta water boiling
+                    over.
+                </p>
+                <strong>‘As Big as It Gets’</strong>
+                <p>n our own universe, it would serve as a window into the forces operating at energies forever beyond
```
Comment (heyapricot) [inline index.html:118]: You missed an `I` at the beginning of the paragraph content

File: index.html [hunk 13]
```diff
@@ -0,0 +1,484 @@
...
+                    Cosmic Extragalactic Polarization, reported their results in a scientific briefing at the Center
+                    for Astrophysics here on Monday and in a set of papers submitted to The Astrophysical Journal.
+                </p>
+                <strong>The Theory of Inflation</strong>
+                <br>
+                <small>Astronomers have found evidence to support the theory of inflation, which explains how the
+                    universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years
+                    ago.
+                </small>
+                <div class="grid-cards">
```
Comment (heyapricot) [inline index.html:143]: Awesome job replicating this 🥇

File: index.html [hunk 14]
```diff
@@ -0,0 +1,484 @@
...
+                    Eschewing email for fear of a leak, Dr. Kovac personally delivered drafts of his work to a select
+                    few, meeting with Dr. Guth, who is now a professor at Massachusetts Institute of Technology (as is
+                    his son, Larry, who was sleeping that night in 1979), in his office last week.</p>
+                <p>“It was a very special moment, and one we took very seriously as scientists,” said Dr. Kovac, who
+                    chose his words as carefully as he tended his radio telescopes.</p>
+                <p>Andrei Linde of Stanford, a prolific theorist who first described the most popular variant of
+                    inflation, known as chaotic inflation, in 1983, was about to go on vacation in the Caribbean last
+                    week when Chao-Lin Kuo, a Stanford colleague and a member of Dr. Kovac’s team, knocked on his door
+                    with a bottle of Champagne to tell him the news.</p>
+                <img src="https://static01.nyt.com/images/2014/03/18/science/space/18cosmos-2/18cosmos-2-jumbo.jpg?quality=90&auto=webp"
```
Comment (heyapricot) [inline index.html:244]: Why is this duplicated?

**Review by heyapricot — APPROVED** [review]
#  Great job completing this project. Looks very nice.
Please address the comments and after that, you are ready to move on. No need from my side for further review.
Modifying the top bar when you scroll was a nice extra touch. Also, great work replicating the Theory of inflation section.
Additional petition: It would be nice if the links in the footer are actual links and not just text elements. 
Tips:
For every HTML/CSS projects please check your files with the [W3 Validator](https://validator.w3.org) and the [CSS Linter](http://csslint.net/)

File: index.html (hunk 1 above)
Comment (htuerker) [reply index.html:46]: it's a thing we used it from original website otherwise it's not working.

File: index.html (hunk 2 above)
Comment (htuerker) [reply index.html:209]: we used youtube's embedded code here.

File: css/styles.css (hunk 11 above)
Comment (htuerker) [reply css/styles.css:217]: Yes, we needed to do it. Because it needs a default option.

File: index.html (hunk 14 above)
Comment (htuerker) [reply index.html:244]: Reusability purposes :)

**Pull request merged**